packages/manager/modules/pci/src/projects/project/storages/databases/databases.routing.js

```diff
@@ -37,6 +42,20 @@ export default /* @ngInject */ ($stateProvider) => {
         DatabaseService.getAllDatabases(projectId).then((databases) =>
           $q.all(map(databases, (database) => getDatabaseObject(database))),
         ),
+      showPaymentWarning: /* @ngInject */ () => (databases) => {
+        const oldDb = databases.find(
+          (db) =>
+            SHELL_NAMES[db.engine] &&
+            SHELL_NAMES[db.engine] !== 'mongo' &&
+            moment(db.createdAt).isBefore(
+              moment(WARNING_DATE.CREATION_DATE, 'dd/MM/YYYY'),
```

Comment:
Nitpick: can you extract date format string to packages/manager/modules/pci/src/components/project/warning-message/warning.constants.js.